src/Compilers/Core/Portable/Microsoft.CodeAnalysis.csproj

```diff
@@ -20,9 +20,6 @@
       Do not install this package manually, it will be added as a prerequisite by other packages that require it.
     </PackageDescription>
   </PropertyGroup>
-
-  <ItemGroup>
-  </ItemGroup>
```

Comment:
It looks like this is the only change under compilers and it is not really related to the stated goal of this PR. Consider reverting this file.

Reply:
When i do, VS changes it back..  def annoying to keep either reverting, or keeping out of this PR.

Reply:
> When i do, VS changes it back..  def annoying to keep either reverting, or keeping out of this PR.  

If you commit only staged files and never stage the file it really doesn't matter what VS does to it locally

---
In reply to: [396792564](https://github.com/dotnet/roslyn/pull/42654#discussion_r396792564) [](ancestors = 396792564)

Reply:
right, but that makes things much less convenient.  now, on every PR in every branch, i have to be extremey careful to not add this file in.  If we just let the change go in, then everything is easy in all branches...

Reply:
> If we just let the change go in, then everything is easy in all branches...

You will need two sign-offs from the compiler team then.

Reply:
WFM.  @dotnet/roslyn-compiler  Can i get two eyes on this totally trivial change to `Microsoft.CodeAnalysis.csproj`?  Thanks!